hydra-cardano-api/src/Hydra/Cardano/Api/TxOut.hs

```diff
@@ -115,13 +115,13 @@ toLedgerTxOut =
 -- Plutus addresses are stripped off it.
 fromPlutusTxOut ::
   (HasMultiAsset era, HasScriptData era, HasInlineDatums era, IsShelleyBasedEra era) =>
-  Network ->
+  NetworkId ->
```

Comment:
Should: Push the smaller `Network` type even further up.

There is no point in requesting the full `NetworkId` here. So let's do the same as before and push this even further up in the call stack.

Reply:
I've pushed it further and now I convert it just before calling `deserializeCommit` in `observeCommitTx`.